autograder/cli/submission/fetch-attempts.py

```diff
@@ -0,0 +1,43 @@
+import os
+import sys
+
+import autograder.api.submission.fetchattempts
+import autograder.cli.submission.common
+
+def run(arguments):
+    result = autograder.api.submission.fetchattempts.send(arguments, exit_on_error = True)
+
+    if (not result['found-user']):
+        print("No matching user found.")
+        return 1
+
+    if (len(result['grading-results']) == 0):
+        print("No attempts found.")
+        return 1
```

Comment:
Return 2 here.
We just want to return different exit status so they can be differentiated in the terminal.